capstone/src/main/java/com/step902020/capstone/Review.java

```diff
@@ -14,32 +21,99 @@
 
   long timestamp;
 
-  @Field(name="name")
-  String name;
+  int likes;
+
+  String individualName;
+
+  String individualEmail;
```

Comment:
Should those be fields?

Reply:
You only need to mark something as `@Field` when you want to change the default `name`. BTW, `text` field below doesn't need `@Field`.